### PR TITLE
Fix LLPC_ASSERT in PatchInout pass phase

### PR DESCRIPTION
**Pull request by jiaolu**

### CODE REVIEW

File: patch/llpcPatchInOutImportExport.cpp [hunk 1]
```diff
@@ -431,25 +431,46 @@ void PatchInOutImportExport::visitCallInst(
                 }
             case ShaderStageTessControl:
                 {
-                    LLPC_ASSERT(callInst.getNumArgOperands() == 3);
-                    Value* pElemIdx = IsDontCareValue(callInst.getOperand(1)) ? nullptr : callInst.getOperand(1);
-                    Value* pVertexIdx = IsDontCareValue(callInst.getOperand(2)) ? nullptr : callInst.getOperand(2);
+                    // Builtin Call has different number of operands
+                    Value* pElemIdx = nullptr; Value* pVertexIdx = nullptr;
```
Comment (amdrexu) [inline patch/llpcPatchInOutImportExport.cpp]: Value* pElemIdx = nullptr; Value* pVertexIdx = nullptr;
->
Value* pElemIdx = nullptr;
Value* pVertexIdx = nullptr;

Two lines

File: patch/llpcPatchInOutImportExport.cpp [hunk 2]
```diff
@@ -431,25 +431,46 @@ void PatchInOutImportExport::visitCallInst(
...
                     pInput = PatchTcsBuiltInInputImport(pInputTy, builtInId, pElemIdx, pVertexIdx, &callInst);
                     break;
                 }
             case ShaderStageTessEval:
                 {
-                    LLPC_ASSERT(callInst.getNumArgOperands() == 3);
-                    Value* pElemIdx = IsDontCareValue(callInst.getOperand(1)) ? nullptr : callInst.getOperand(1);
-                    Value* pVertexIdx = IsDontCareValue(callInst.getOperand(2)) ? nullptr : callInst.getOperand(2);
+                    // Builtin Call has different number of operands
+                    Value *pElemIdx = nullptr; Value* pVertexIdx = nullptr;
```
Comment (amdrexu) [inline patch/llpcPatchInOutImportExport.cpp]: Value *pElemIdx = nullptr; Value* pVertexIdx = nullptr;
->
Value *pElemIdx = nullptr;
Value* pVertexIdx = nullptr;

Two lines

**Comment by amdvlk-admin** [comment]
### Test summary for commit 62910005a84d6bb8b0edb5e4069c01714afdea35
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev 9a1f99f049f3b417242dc7c09b86c52fd7d9ceaf</li>
<li>SPVGEN: dev 64013f150c2d41965a3d36fa159b55fba52b727a</li>
<li>XGL: dev 4d91a4045078cf6884ee05007eac9eec2b881dbb</li>
<li>LLVM: amd-vulkan-dev 4305f776f1207189667a383e22661ccd1e99cd25</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 900/900 (100.0%)</li>
<li>Failed: 0/900 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 162031/162035 (100.0%)</li>
<li><details><summary>Failed: 4/162035 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.push_constant.graphics_pipeline.count_3_shaders_vert_geom_frag
Stack trace: Image mismatch

FAILURE: dEQP-VK.pipeline.push_constant.graphics_pipeline.overlap_3_shaders_vert_geom_frag
Stack trace: Image mismatch

FAILURE: dEQP-VK.pipeline.push_constant.graphics_pipeline.overlap_5_shaders_vert_tess_geom_frag
Stack trace: Image mismatch

FAILURE: dEQP-VK.pipeline.spec_constant.graphics.geometry.default_value.bool
Stack trace: Timeout or crahsed
```
</p></details></li>


</ul></details>


</ul></details>

**Comment by JacobHeAMD** [comment]
Hi @jiaolu , please rebase your commit to the latest dev, it will cause Vega10 agent of Jenkins hang otherwise.

**Comment by amdvlk-admin** [comment]
### Test summary for commit a331a2fff059e9a81a40d4bf995970ff7c80df51
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev 9a1f99f049f3b417242dc7c09b86c52fd7d9ceaf</li>
<li>SPVGEN: dev 64013f150c2d41965a3d36fa159b55fba52b727a</li>
<li>XGL: dev 4d91a4045078cf6884ee05007eac9eec2b881dbb</li>
<li>LLVM: amd-vulkan-dev 4305f776f1207189667a383e22661ccd1e99cd25</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 900/900 (100.0%)</li>
<li>Failed: 0/900 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**